Sources/LibMobileCoin/McRngCallback.swift

```diff
@@ -5,15 +5,34 @@
 import Foundation
 import LibMobileCoin
 
+func withMcRngObjCallback<T>(
```

Comment:
some thoughts on this change:

if we're moving all RNG behavior to MobileCoinRng sub-classes, then the "rng function" that accepts the context will be the same because it relies on common behavior of the base class. Given that, I don't think it needs to be parameterized here, rather it can just be set explicitly.